### PR TITLE
Upload minified files to CircleCI build artifacts

### PR DESCRIPTION
**Pull request by ylemkimon**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -75,10 +75,10 @@ jobs:
 
       - store_artifacts:
           path: dist/katex.js
-          destination: katex.js
+          destination: katex.min.js
```
Comment (edemaine) [inline .circleci/config.yml]: The sources and destinations have mismatching filenames.  Did you mean `dist/katex.min.js`?  Ditto below

Comment (ylemkimon) [reply .circleci/config.yml]: @edemaine Yes, thank you!

**Comment by ylemkimon** [comment]
This is thousandth commit!

**Comment by kevinbarabash** [comment]
I could've sworn it was in the 700s last time I looked. 😅

**Comment by kevinbarabash** [comment]
@ylemkimon are we using these artifacts for anything at the moment or do you have future plans for them?

**Comment by ylemkimon** [comment]
@kevinbarabash No, we're not using them for now, and I think it's highly unlikely we'll use them in the future, as we have Netlify builds. I've removed them in #1658.

**Pull request merged**